crates/sui-bridge/src/server/mod.rs

```diff
@@ -58,22 +60,31 @@ pub const ADD_TOKENS_ON_SUI_PATH: &str =
 pub const ADD_TOKENS_ON_EVM_PATH: &str =
     "/sign/add_tokens_on_evm/:chain_id/:nonce/:native/:token_ids/:token_addresses/:token_sui_decimals/:token_prices";
 
-/// BridgeNode's public metadata that is acceesible via the `/ping` endpoint.
+// BridgeNode's public metadata that is accessible via the `/ping` endpoint.
 // Be careful with what to put here, as it is public.
-#[derive(serde::Serialize, Clone)]
+#[derive(serde::Serialize)]
 pub struct BridgeNodePublicMetadata {
     pub version: Option<String>,
+    pub network_key_pair: Option<Arc<NetworkKeyPair>>,
```

Comment:
a few thoughts:
1. hmm i don't quite rmr why version is `Option<String>` as opposed to `String`... but i think we don't need `Option` for `network_key_pair`?
2. let's avoid storing the keypair here to reduce attack surface, we can just store the public key bytes.
3. how about calling it `metrics_pubkey`?
4. we may need to think about versioning of this endpoint but i'm comfortable with directly adding a new field given this endpoint is not widely used yet. Just be aware that when it queries pubkey in sui-proxy, we need to tolerate ser/de errors because of versions

Reply:
> let's avoid storing the keypair here to reduce attack surface, we can just store the public key bytes.

Whoops, good point. I've changed things so that the network key is generated in the `BridgeNodeConfig` (could be optionally overridden by operators if they want), and just the public key is passed to `BridgeNodePublicMetadata`

> hmm i don't quite rmr why version is Option<String> as opposed to String... but i think we don't need Option for network_key_pair?

I left it as an Option so I could continue to use the test util, `BridgeNodePublicMetadata::empty_for_testing()`, rather than generate a stub `Ed25519PublicKey` in the tests.

> we may need to think about versioning of this endpoint but i'm comfortable with directly adding a new field given this endpoint is not widely used yet. Just be aware that when it queries pubkey in sui-proxy, we need to tolerate ser/de errors because of versions

we can definitely make sui-proxy tolerate errors when hitting the `/metrics/network_key` endpoint, I'm happy to change it to `/v1/metrics_pub_key` or something like that?